scraper.rb

```diff
@@ -141,7 +151,7 @@ def state
   SearchPage.new(url).to_h[:members]
 end.uniq
 
-members.select { |m| m[:name].start_with? 'Hon' }.each do |mem|
+members.each do |mem|
   person = MemberPage.new(mem[:url])
-  ScraperWiki.save_sqlite(%i(id name term), person.to_h)
+  ScraperWiki.save_sqlite(%i(id name term chamber), person.to_h)
```

Comment:
Defensive, although quite possibly mistakenly - I assumed that the `id` column wasn't unique on its own since `name` and `term` were already in the key, so probably `chamber` should be there too now that there are multiple chambers. I'm happy to remove it if that doesn't make sense.